cmd/internal/cli/startvm.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+)
+
+func prepareVM(cmd *cobra.Command, image string, args []string) {
```

Comment:
I think this function can be renamed `execVM()` to better align with `execStarter()`. Also, let's move this into `actions.go`, since it's common across architectures.